chart/values.yaml

```diff
@@ -300,7 +300,7 @@ components:
       minAgePrebuildDays: 7
       contentRetentionPeriodDays: 21
       contentChunkLimit: 1000
-    definitelyGpDisabled: "false"
+    definitelyGpDisabled: true
```

Comment:
Is this ok changing the type from a string to a boolean?

Reply:
Yep, Confirmed that we aren't using it anywhere to toggle something, but [just passing it down into the `server-configmap`, and follow the same pattern there like other boolean values ](https://github.com/gitpod-io/gitpod/blob/54a765d321236ef42dceeb154cf477b17a8aaf32/chart/templates/server-configmap.yaml#L38)(soemthing like `enableLocalApp`)